package.json

```diff
@@ -15,11 +15,14 @@
     "autoprefixer": "10.4.14",
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.7",
-    "next": "13.4.7",
+    "next": "^13.4.7",
     "postcss": "8.4.24",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
❌  ❓ 
why did you change these?

Reply:
For some reason, when I run '**npm run dev**', I encounter the error message 'next' is not recognized as an internal or external command. Upon searching for a solution to this error, I found that the recommended solution is to run 'npm install react react-dom next'.

Reply:
then you should not commit them,
if you want to do Es-lint fixes do them in a seperated commit

Reply:
check out this to know what this character means
https://stackoverflow.com/a/22345808